internal/resources/kubernetessecret/resource_secret_test.go

```diff
@@ -55,15 +55,14 @@ func TestAcceptanceForSecretResource(t *testing.T) {
 	testConfig := testGetDefaultAcceptanceConfig(t)
 
 	// If the flag to execute kubernetes secret tests is not found, run this as a mock test by setting up an http intercept for each endpoint.
-	if _, found := os.LookupEnv("ENABLE_SECRET_ENV_TEST"); !found {
+	_, found := os.LookupEnv("ENABLE_SECRET_ENV_TEST")
+	if !found {
 		os.Setenv("TF_ACC", "true")
 		os.Setenv("TMC_ENDPOINT", "dummy.tmc.mock.vmware.com")
 		os.Setenv("VMW_CLOUD_API_TOKEN", "dummy")
 		os.Setenv("VMW_CLOUD_ENDPOINT", "console.cloud.vmware.com")
 
 		log.Println("Setting up the mock endpoints...")
-
-		testConfig.setupHTTPMocks(t)
```

Comment:
Why is this function call deleted?

Reply:
@shobha2626 I see that call is moved to steps `testConfig.setupHTTPMocks(t, DockerSecretType)`